pdmaint

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/Users/mberglof/.virtualenvs/pdmaint/bin/python
```

Comment:
Could you change this back to `#!/usr/bin/env python` then update your pull request?

Reply:
@mberglof When you have the chance, ☝️please and thanks!

Reply:
Since this is an easy enough thing to fix and the new options appear to work as expected I think I'll just merge this then make the fix myself...